package.json

```diff
@@ -30,7 +30,6 @@
   },
   "homepage": "https://github.com/dhis2/d2",
   "dependencies": {
```

Comment:
Out of all the deps in this hash only the `whatwg-fetch` makes sense as a runtime dep. Would you mind moving the others as well?